package.json

```diff
@@ -115,12 +119,20 @@
   },
   "bundlesize": [
     {
-      "path": "./dist/vue-instantsearch.js",
+      "path": "./dist/vue2/umd/index.js",
```

Comment:
changing existing file names could be a breaking change for when someone is forcing UMD at the moment, could we keep the existing names the same and for example only add a new folder for vue 3?

Reply:
Actually no, let' have a clean dist and release this as a major version

Reply:
Oh I didn't realize we don't have jsdelivr or unpkg in the package.json at Vue InstantSearch. Yeah, the url of UMD will change. Let's go as a breaking change.